src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -50,7 +50,9 @@
        HelixJobProperties- Must be JSON.  String describing Helix MC-specific metadata.       Default: <Blank>
        HelixArchLabel    - If HelixJobProperties is not set, we'll use this to fill it out    Default: <Blank>
        HelixConfigLabel  - If HelixJobProperties is not set, we'll use this to fill it out    Default: <Blank>
-       
+       SecondaryQueue    - When submitting multi-stage tests, queue for secondary jobs        Default: <Blank>
```

Comment:
Is secondary the right name here? It makes me feel like it would only be able to string together two jobs. If that is true then secondary is probably OK, but if it essentially allows linking of of as many necessary maybe "Next" or "Continuation" queue might be better names.